.travis.yml

```diff
@@ -7,6 +7,20 @@ script:
 
 matrix:
   include:
+    - name: coverity
+      script:
+        - echo "coverity scan script"
+      install:
+        - echo "coverity scan install"
+      addons:
+        coverity_scan:
+          project:
+            name: "airsonic/airsonic"
```

Comment:
Should this be `airsonic/airsonic` or should it be just `airsonic`?

Reply:
Ah, I think it might be this way because it includes the user part of the github project?  In that case nevermind.

Reply:
It [includes]( https://scan.coverity.com/travis_ci ) the user part :)